packages/p2p/src/components/my-ads/edit-ad-form-payment-methods.jsx

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react';
 import { observer } from 'mobx-react-lite';
+import { useP2PAdvertiserPaymentMethods } from '@deriv/hooks';
+import { localize } from 'Components/i18next';
 import { useStores } from 'Stores';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
 import PaymentMethodCard from '../my-profile/payment-methods/payment-method-card';
-import { localize } from 'Components/i18next';
 import BuyAdPaymentMethodsList from './buy-ad-payment-methods-list.jsx';
 import SellAdPaymentMethodsList from './sell-ad-payment-methods-list.jsx';
-import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
 
 const EditAdFormPaymentMethods = ({ is_sell_advert, selected_methods, setSelectedMethods, touched }) => {
-    const { my_ads_store, my_profile_store } = useStores();
+    const { my_ads_store } = useStores();
+    const { data: advertiser_payment_methods } = useP2PAdvertiserPaymentMethods();
```

Comment:
shouldn't we remove the hook call from the child component then? '`sell-ad-payment-methods-list.jsx`'?

Reply:
we still need it here, or we can pass it as props instead

Reply:
yeah.. if not removed from the child components wont it trigger multiple api calls